lib/fb/page.rb

```diff
@@ -30,12 +35,9 @@ def initialize(options = {})
     #   @name="page_fans", @description="Weekly: The
     #   number of new people who have liked your Page (Unique Users)",
     #   @value=123>,..}
-    def insights
+    def insights(options = {})
       @insights ||= begin
```

Comment:
I think you are right, in this case you better not memoize because the return value will be decided by `options`